runtime/src/lib.rs

```diff
@@ -817,41 +824,38 @@ impl cumulus_pallet_xcmp_queue::Config for Runtime {
 }
```

Comment:
does it need `type VersionWrapper = ();` or is that in newer version ? https://github.com/substrate-developer-hub/substrate-parachain-template/blob/master/runtime/src/lib.rs#L428

Reply:
@ltfschoen You are pointing to the latest master branch. I followed   https://github.com/substrate-developer-hub/substrate-parachain-template/blob/polkadot-v0.9.9/runtime/src/lib.rs#L430